rust/Dockerfile

```diff
@@ -1,20 +1,10 @@
-FROM lukemathwalker/cargo-chef:latest AS chef
-WORKDIR app
-
-FROM chef AS planner
-COPY . .
-RUN cargo chef prepare --recipe-path recipe.json
-
-FROM chef AS builder 
-COPY --from=planner /app/recipe.json recipe.json
-RUN cargo chef cook --release --recipe-path recipe.json
-COPY . .
-RUN cargo build --release 
-
-
+FROM rust:1.66.0 as build-env
+WORKDIR /app
+COPY . /app
+RUN cargo build --release
 
 FROM gcr.io/distroless/cc
-WORKDIR /app
-COPY --from=builder /app/target/release/smart_bot ./
-EXPOSE 8001
-CMD ["/app/smart_bot"]
\ No newline at end of file
+COPY --from=build-env /app/target/release/a-29-from-scratch smart_bot
+
+EXPOSE 8001/tcp
```

Comment:
@krischal111 Did you submit docker with this code? Please expose `8001` as the endpoint.